lago/cmd.py

```diff
@@ -760,6 +759,79 @@ def do_collect(prefix, output, no_skip, **kwargs):
 def do_deploy(prefix, **kwargs):
     prefix.deploy()
 
+@lago.plugins.cli.cli_plugin(
+    help='Verify that the machine runninh Lago is well configured and configure if needed'
+)
+@lago.plugins.cli.cli_plugin_add_argument(
+    '--username',
+    '-u',
+    help='Which user needs to be configured',
+    action='store',
+)
+
+@lago.plugins.cli.cli_plugin_add_argument(
+    '--envs-dir',
+    '-e',
+    dest='envs_dir',
+    help='Which directory the qemu has access permissions',
+    default="/var/lib/lago",
+    type=os.path.abspath,
+    action='store',
+)
+
+@lago.plugins.cli.cli_plugin_add_argument(
+    '--verify',
+    '-v',
+    help='Return report that describes which configurations are OK, and which are not.',
+    action='store_true',
+)
+@in_lago_prefix
+@with_logging
+def do_setup(
+    prefix, username, envs_dir, verify, **kwargs
+):
+    msg_error = []
+    if (username):
+        if (check_user(username)):
+            msg_error.append("Username doesn't exists " + username)
+    else:
+        username = os.getenv("SUDO_USER") or os.getenv("USER") 
+
+    if (envs_dir):
+        if (check_directory(envs_dir)):
+            msg_error.append("Directory doesn't exists "+ envs_dir)
+
+    if ( msg_error ):
+        msg_error_str = '\n'.join(msg_error)
+        LOGGER.error("%s", msg_error_str)
+        sys.exit(1)
+
+    config_dict = check_configuration(username,envs_dir)
+    (verify_status,list_not_configure) = validate_status(config_dict)           
+    verify_lago = VerifyLagoStatus(username,envs_dir,config_dict,verify_status)
+    
+    if (verify):
+       verify_lago.displayLagoStatus(True)
+       if verify_status:
+              sys.exit(0)
+       else:
+              sys.exit(2)   
+    else:
+       if (os.getuid() != 0): 
+          print("Please use 'sudo', you need adminstrator permissions for configuration")
+          sys.exit(1)
+       else:
```

Comment:
There is no need for an `else` here. If the first condition was true, the program exited.